paper-icon-button.html

```diff
@@ -103,7 +117,8 @@
     is: 'paper-icon-button',
 
     behaviors: [
-      Polymer.PaperButtonBehavior
+      Polymer.PaperButtonBehavior,
+      Polymer.PaperRadioButtonBehavior
```

Comment:
It gives it an inky ripple on focus, as currently this element has no focus style. Alternatively, if we don't want to use the ripple, we can add a background style, but it won't be as bouncy.

The added behaviour only has an action for when the focus changes because of keyboard events.